src/content.config.js

```diff
@@ -0,0 +1,32 @@
+import { defineCollection, z } from 'astro:content';
+import { glob, file } from 'astro/loaders'; // Not available with legacy API
+
+const talks = defineCollection({
+  loader: glob({ pattern: "**/*.md", base: "./src/content/talks" }),
+  schema: z.object({
+    tracks: z.array(z.string()),
+    title: z.string(),
+    level: z.string(),
+    tags: z.array(z.string()),
+    duration: z.number(),
+    day: z.string(),
+  })
+});
```

Comment:
Whoa this is strange stuff. I wonder whether they have some typescript API too. But looks good!